source/dpq/attributes.d

```diff
@@ -15,10 +15,7 @@ version(unittest) import std.stdio;
    for Connection's ORM functions. If none is set, dpq will
    default to the structure's name in lower_snake_case.
  */
-RelationAttribute relation(string name)
-{
-   return RelationAttribute(name);
-}
```

Comment:
Since `@relation` UDA was actually a function, `hasUDA!(T, RelationAttribute)` alway returned `false`, because symbols with `@relation` indeed did not aquired `RelationAttribute`. The first solution that comes in mind is using `alias`, but `relation` was a function for a reason and `alias` won't work that simple.